CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
-        public Guid CompanyId { get; set; }
```

Comment:
The `[ForeignKey]` attribute should precede its navigation property target (`Company`), not the foreign key field. This usage has no effect here and can be safely removed.

```suggestion
public Guid CompanyId { get; set; }
```